### DISCUSSION TITLE
Google Auth/Supabase Not Working in Production (Vercel)

### DISCUSSION THREAD

**Discussion by yachty66** [supabase/supabase#28883] [Questions]
I'm encountering an issue with Google authentication using Supabase in my production environment on Vercel, while it works fine locally. Here are the key details:

- The `signInWithGoogle` function is not being called in production.
- Local development environment works as expected.
- Deployed on Vercel
- Using Supabase for authentication
- Google Sign-In is enabled in Supabase project settings
- Callback URLs and authorized origins are set up in both Supabase and Google Console

I've already tried the following:
- Verified environment variables in Vercel deployment
- Double-checked all redirect URIs and authorized origins
- Added console logs throughout the authentication flow
- Ensured CORS is properly configured

Despite these efforts, the authentication process doesn't seem to initiate in the production environment. The `signInWithGoogle` function is never reached.

Has anyone encountered a similar issue or can suggest additional troubleshooting steps? Any insights would be greatly appreciated.

**Comment by adrknapp** [comment]
Have you fixed it? Facing the same here
